tests/app/clients/test_firetext.py

```diff
@@ -47,7 +47,7 @@ def test_send_sms_successful_returns_firetext_response(mocker, mock_firetext_cli
 
     with requests_mock.Mocker() as request_mock:
         request_mock.post('https://example.com/firetext', json=response_dict, status_code=200)
-        response = mock_firetext_client.send_sms(to, content, reference)
+        response = mock_firetext_client.send_sms(to, content, reference, False)
```

Comment:
```suggestion
        response = mock_firetext_client.send_sms(to, content, reference, international=False)
```

Minor: this should make it easier for someone in future (avoids them having to find and read the function signature).